src/api/route-services/gem2s-response.js

```diff
@@ -38,21 +42,14 @@ const gem2sResponse = async (io, message) => {
   // Fail hard if there was an error.
   await validateRequest(message, 'GEM2SResponse.v1.yaml');
 
-  const { experimentId } = message;
-
-  if (!message.table) {
-    await sendUpdateToSubscribed(experimentId, message, io);
-    return;
-  }
-
-  const { item, table: tableName } = message;
+  const {
+    experimentId, taskName, item,
+  } = message;
 
-  if (tableName.includes('experiments')) {
-    await experimentService.updateExperiment(experimentId, item);
-  } else if (tableName.includes('samples')) {
-    const { projectUuid } = item;
-    await samplesService.updateSamples(projectUuid, item);
-  }
+  pipelineHook.run(taskName, {
+    experimentId,
+    item,
+  });
```

Comment:
this needs to be `await pipelineHook.run`

Reply:
added